makefile.shade

```diff
@@ -32,22 +32,15 @@ use-standard-lifecycle
 
         foreach (var package in Directory.EnumerateDirectories(PACKAGE_DROP_SHARE))
         {
-            var versionsToDelete = Directory.EnumerateDirectories(package)
-                                            .OrderByDescending(d => d, StringComparer.OrdinalIgnoreCase)
-                                            .Skip(3);
-
-            foreach (var versionDir in versionsToDelete)
-            {
-                Log.Info("Deleteing " + versionDir);
-                try
-                {
-                    Directory.Delete(versionDir, recursive: true);
-                }
-                catch (IOException)
-                {
-                    // Ignore.
-                }
-            }
+          // delete every thing 
```

Comment:
Change the target to `publish`. We can't delete everything prior to knowing this is a coherent build.